frontend/src/metabase/query_builder/components/dataref/DatabaseTablesPane/DatabaseTablesPane.tsx

```diff
@@ -0,0 +1,107 @@
+import React, { useMemo } from "react";
+import { ngettext, msgid } from "ttag";
+import _ from "underscore";
+
+import Tables from "metabase/entities/tables";
+import Questions from "metabase/entities/questions";
+import type { Card } from "metabase-types/api";
+import type { State } from "metabase-types/store";
+import Database from "metabase-lib/lib/metadata/Database";
+import {
+  NodeListItemLink,
+  NodeListItemName,
+  NodeListItemIcon,
+  NodeListTitle,
+  NodeListContainer,
+  NodeListIcon,
+  NodeListTitleText,
+} from "../NodeList.styled";
+import { ModelId } from "./DatabaseTablesPane.styled";
+
+interface DatabaseTablesPaneProps {
+  show: (type: string, item: unknown) => void;
+  database: Database;
+  models: Card[];
+}
+
+const DatabaseTablesPane = ({
+  database,
+  show,
+  models,
+}: DatabaseTablesPaneProps) => {
+  const tables = useMemo(
+    () => database.tables.sort((a, b) => a.name.localeCompare(b.name)),
+    [database.tables],
+  );
+  models = useMemo(
```

Comment:
nit: overwriting parameters should be catched by eslint, but for some reason it didn't do it here. Could there be another variable, e.g. `sortedModels`?

Reply:
Done https://github.com/metabase/metabase/pull/25494/commits/daef5ce1d85a2f32b721859323d155548a0131c9